src/lightning_lite/lite.py

```diff
@@ -1,3 +0,0 @@
-class LightningLite:
```

Comment:
This file has to be removed, otherwise, when we move the Lite class here in the next PR, it won't show up as a clean file move.